cmd/vpp-agent-ctl/data_cmd.go

```diff
@@ -314,7 +314,7 @@ func (ctl *VppAgentCtl) createTap() {
 				//	InterfaceWithIP: "memif1",
 				//},
 				Tap: &interfaces.Interfaces_Interface_Tap{
-					HostIfName: "tap1",
+					HostIfName: "tap-host",
```

Comment:
This change doesn't affect any other commands, like Vlado mentioned earlier?

Reply:
Without this change it doesn't work, as hostname for tap and ltap are not the same and so tap is not configured correctly. How can I verify whether it affect other commands?

Reply:
Actually, the name of the interface `tap1` stayed the same, so hopefully nothing else will be affected since most other referings are through the `Name` and not `HostIfName`.